public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -32,164 +33,143 @@ include ../_quickstart_repo
     .file typings.json
 :marked
   ## Keep the app transpiling and running
-  We want to start the TypeScript compiler, have it watch for changes, and start our server. We'll do this by typing
+  To start the TypeScript compiler in watch mode and start the server, type the following:
+  
 
 code-example(language="bash").
   npm start
 
 :marked
-  This command runs the compiler in watch mode, starts the server, launches the app in a browser,
-  and keeps the app running while we continue to build the Tour of Heroes.
+  This command launches the app in a browser
+  and keeps the app running while you build the Tour of Heroes.
 
 .l-main-section
   :marked
-    ## Show our Hero
-    We want to display Hero data in our app
-
-    Let's add two properties to our `AppComponent`, a `title` property for the application name and a `hero` property
-    for a hero named "Windstorm".
+    ## Show your hero
+    To display hero data in the app,
+    add two properties to the `AppComponent`: a `title` property for the app name and a `hero` property
+    for a hero named "Windstorm."
 
   +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'app-component-1', 'app.component.ts (AppComponent class)')(format=".")
 
   :marked
-    Now we update the template in the `@Component` decoration with data bindings to these new properties.
+    Now update the template in the `@Component` decoration with data bindings to these new properties.
 
   +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero')
 
   :marked
-    The browser should refresh and display our title and hero.
+    The browser refreshes and display the title and hero name.
 
-    The double curly braces tell our app to read the `title` and `hero` properties from the component and render them.
+    The double curly braces instruct the app to read the `title` and `hero` properties from the component and render them.
     This is the "interpolation" form of one-way data binding.
 .l-sub-section
   :marked
-    Learn more about interpolation in the [Displaying Data chapter](../guide/displaying-data.html).
+    Learn more about interpolation in the [Displaying Data page](../guide/displaying-data.html).
 :marked
   ### Hero object
 
-  At the moment, our hero is just a name.  Our hero needs more properties.
+  Our hero needs more properties.
```

Comment:
Do a global search for "our", "we", "let's", and "us". You want to get rid of all first person.